interfaces/builtin/nfs_mount_test.go

```diff
@@ -0,0 +1,100 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016-2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/builtin"
+	"github.com/snapcore/snapd/interfaces/seccomp"
+	"github.com/snapcore/snapd/snap"
+	"github.com/snapcore/snapd/testutil"
+)
+
+type NfsMountInterfaceSuite struct {
+	iface    interfaces.Interface
+	slotInfo *snap.SlotInfo
+	slot     *interfaces.ConnectedSlot
+	plugInfo *snap.PlugInfo
+	plug     *interfaces.ConnectedPlug
+}
+
+const nfsMountConsumerYaml = `name: consumer
+version: 0
+apps:
+ app:
+  plugs: [nfs-mount]
+`
+
+const nfsMountCoreYaml = `name: core
+version: 0
+type: os
+slots:
+  nfs-mount:
+`
+
+var _ = Suite(&NfsMountInterfaceSuite{
+	iface: builtin.MustInterface("nfs-mount"),
+})
+
+func (s *NfsMountInterfaceSuite) SetUpTest(c *C) {
+	s.plug, s.plugInfo = MockConnectedPlug(c, nfsMountConsumerYaml, nil, "nfs-mount")
+	s.slot, s.slotInfo = MockConnectedSlot(c, nfsMountCoreYaml, nil, "nfs-mount")
+}
+
+func (s *NfsMountInterfaceSuite) TestName(c *C) {
+	c.Assert(s.iface.Name(), Equals, "nfs-mount")
+}
+
+func (s *NfsMountInterfaceSuite) TestSanitizeSlot(c *C) {
+	c.Assert(interfaces.BeforePrepareSlot(s.iface, s.slotInfo), IsNil)
+}
+
+func (s *NfsMountInterfaceSuite) TestSanitizePlug(c *C) {
+	c.Assert(interfaces.BeforePreparePlug(s.iface, s.plugInfo), IsNil)
+}
+
+func (s *NfsMountInterfaceSuite) TestAppArmorSpec(c *C) {
+	spec := &apparmor.Specification{}
```

Comment:
This is no longer allowed. Sadly we require a non-zero value since d8a2e847e507caae0d7162ffb5a46fb8186a376f.

Reply:
Fixed